### PR TITLE
Addition of Read_Capacity_10, Start/Stop, and fix misc bugs

### PR DESCRIPTION
**Pull request by bgly**

### CODE REVIEW

**Comment by bgly** [comment]
Please Review @agrover @famz

**Comment by agrover** [comment]
Looks good to me. Seems pretty straightforward but I'll give @famz 24 hours to review before merging.

(FWIW next time you can reuse the existing PR by using `git push --force` to the the existing pr's branch)

File: api.c [hunk 1]
```diff
@@ -859,3 +859,23 @@ int tcmu_emulate_mode_select(
 
 	return SAM_STAT_GOOD;
 }
+
+int tcmu_emulate_start_stop(struct tcmu_device *dev, uint8_t *cdb,
+			                                uint8_t *sense)
+{
+	if ((cdb[4] >> 4) & 0xf)
```
Comment (bgly) [inline api.c]: Nope just consistency between our implementations

**Comment by bgly** [comment]
@agrover @famz Thanks for reviewing!

**Comment by agrover** [comment]
thanks @bgly and @famz!

**Pull request merged**